tests/common.go

```diff
@@ -4397,6 +4399,66 @@ func collectAndCopyDiagsOnWorkerNodes(issueKey string) {
 	}
 }
 
+func collectAndCopyLogsFromPodsForTest(testCaseName string, podLabel map[string]string, namespace string, logLabel string) {
```

Comment:
Rename this function to just `collectLogsFromPods`
Also add a comment explaining about the function

Reply:
Done.